lib/diagnostic.rb

```diff
@@ -100,13 +103,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.pop(2)'
 
 # What would `arr` equal after:
 arr[arr.length + 2] = 99
 # Write your response as a ruby string
 
-Diagnostic.array_add_to = 'your answer here'
+Diagnostic.array_add_to = [12, 34, nil, 99]
```

Comment:
Missing a `nil` in there

`Diagnostic.array_add_to = [12, 34, nil, nil, 99]` Reason being, you're adding 2 to the length prior to 99 being injected.